sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -39,9 +39,13 @@ public final class StorageBlobClient: PipelineClient {
     /// The global maximum number of managed transfers that will be executed concurrently by all `StorageBlobClient`
     /// instances. The default value is `maxConcurrentTransfersDefaultValue`. To allow this value to be determined
     /// dynamically based on current system conditions, set it to `maxConcurrentTransfersDynamicValue`.
-    public static var maxConcurrentTransfers: Int {
-        get { return manager.maxConcurrency }
-        set { manager.maxConcurrency = newValue }
+    public static var maxConcurrentTransfers: Int? {
```

Comment:
We should never have a situation where this returns nil - we could just remove the type annotation and let `manager` be of type `URLSessionTransferManager` on L68.

Reply:
I pushed 39d22b8 to address this. If you agree with the change, then feel free to merge :)

Reply:
Yup. I dig it!